### PR TITLE
print when a repo was updated

### PR DESCRIPTION
**Pull request by steveoh**
http://gitpython.readthedocs.io/en/stable/reference.html#git.remote.FetchInfo

### CODE REVIEW

File: src/forklift/cli.py [hunk 1]
```diff
@@ -150,7 +150,13 @@ def git_update():
             log.info('git updating: {}'.format(repo_name))
             repo = _get_repo(folder)
             origin = repo.remotes[0]
-            origin.pull()
+            fetch_infos = origin.pull()
+
+            if len(fetch_infos) > 0:
+                if fetch_infos[0].flags == 4:
+                    log.debug('no updates to pallet')
```
Comment (stdavis) [inline src/forklift/cli.py]: Why `debug` here and not `info`?

Comment (steveoh) [inline src/forklift/cli.py]: figured the console didn't care unless it changed

**Comment by stdavis** [comment]
LGTM

**Comment by stdavis** [comment]
Merged.

**Pull request closed without merging**